### ISSUE TITLE
Database Insertion issue

### ISSUE THREAD

**Issue by TheTemportalist** [GawainLynch/bolt-extension-boltforms#4]
When inserting from a multi-choice field to a database, the string 'Array' is set, instead of an value actually signifying the options.
Field:
```yaml
mods:
      type: choice
      options:
        required: true
        label: What do you want to subscribe to?
        choices: [ Origin, Compression ]
        multiple: true
```
Database column is named 'mods', type: varchar(256), utf8_general_ci

**Comment by GawainLynch** [comment]
Fixed in 05acb44a020ba4717c0dac76cd8cda43e6410ab1 

I also did the notification if the form name is missing that you hit on the weekend.  I'll tag a release later today (UTC) when I finish a couple of other tweaks.

**Comment by TheTemportalist** [comment]
cool, thank you
